awx/settings/development.py

```diff
@@ -156,4 +156,6 @@
     socket.gethostbyname('docker.for.mac.internal')
     os.environ['SDB_NOTIFY_HOST'] = 'docker.for.mac.internal'
 except Exception:
-    os.environ['SDB_NOTIFY_HOST'] = os.popen('ip route').read().split(' ')[2]
+    ip_route = os.popen('ip route').read()
+    if ip_route:
+        os.environ['SDB_NOTIFY_HOST'] = ip_route.split(' ')[2]
```

Comment:
I considered adding `del ip_route` here, because it might be a large string, and we also maintain a lot of overhead that scales with the size of the settings data.

Reply:
`ip` is fairly ubiquitous these days; what version of Mac OS X are you on?

Reply:
Also, shouldn't this code be running in a Linux _container_?